src/app/retrospective/retrospective-list/retrospective-list.component.html

```diff
@@ -59,6 +58,65 @@
                         </mat-table>
                     </div>
                 </div>
+                <br>
+                <div *ngIf="isAdmin && !showAll" class="action-buttons">
+                    <button mat-raised-button color="primary" (click)="changeRetrospectiveList()">Show All
+                    </button>
+                </div>
+            </div>
+            <div *ngIf="isAdmin && showAll" class="single-grid">
```

Comment:
Is there any way in which:
1. We can just hide the `Show More/Less` button.
2. Show the button but not the table,

...if there are no other retrospectives?

Reply:
Added a condition to not show the button if there is no other retro.